rent.go

```diff
@@ -22,7 +22,7 @@ var (
 
 	// ZeroRentParas are test parameters for de/serialization using zero vbyte rent cost.
 	// Only use this var in testing. Do not modify.
-	ZeroRentParas = &DeSerializationParameters{RentStructure: &RentStructure{
+	ZeroRentParas = &ProtocolParameters{RentStructure: &RentStructure{
```

Comment:
Isn't the total supply missing here?

Reply:
Maybe we need ZeroRentParasIOTA here?

Reply:
I was thinking the same, but we don't run validation where we use ZeroRentParas, so it being 0 is ok